packages/connect-explorer/src/js/data/methods/management/firmwareUpdate.ts

```diff
@@ -0,0 +1,46 @@
+const name = 'firmwareUpdate';
+const docs = 'methods/firmwareUpdate.md';
```

Comment:
It looks like this variable `docs` is never used.

Reply:
removed. docs does not exist in https://github.com/trezor/connect/tree/develop/docs/methods